### DISCUSSION TITLE
How to get rid of ghosts ?

### DISCUSSION THREAD

**Discussion by Fred-Vatin** [tadashi-aikawa/obsidian-another-quick-switcher#254] [Q&A] [answered]
Hi,

How can I clean up those obsolete references ?
![image](https://github.com/user-attachments/assets/626a4e14-7aa9-421b-9d58-541f5b5330ff)

**Comment by tadashi-aikawa** [comment, marked as answer]
Hi, @Fred-Vatin :)

To remove those references, you need to delete the internal links from the text that point to files that no longer exist. (AQS does not have a feature to do this automatically.)

Using the "Show backlinks" command in the dialog will display a list of sources for the ghost links, making the task a bit easier.

![image](https://github.com/user-attachments/assets/e0d7cd73-914f-402f-8bd4-e32247e130ab)
